### PR TITLE
Implement parser for raw data

### PR DESCRIPTION
**Pull request by Perry26**
The results look very weird, but I don't think thats on me

### CODE REVIEW

File: svelte/src/routes/test/scripts/converter/index.ts [hunk 1]
```diff
@@ -1,45 +1,147 @@
...
+	rootNodes.forEach(function calulateNestingLevels(node: ConvertedNode, level: number = 0) {
+		node.level = level;
+		node.members.forEach(n => calulateNestingLevels(n, level + 1));
+	});
+
+	const links: ConvertedEdge[] = rawData.elements.edges.map(({data}) => {
+		return {
+			id: data.id,
+			source: data.source,
+			target: data.target,
```
Comment (Adityapnn811) [inline svelte/src/routes/test/scripts/converter/index.ts:151]: I think we should add the type too

**Review by Adityapnn811 — COMMENTED** [review]
The rest looks fine to me

File: svelte/src/routes/test/scripts/converter/index.ts (hunk 1 above)
Comment (Perry26) [reply svelte/src/routes/test/scripts/converter/index.ts:151]: Done :)

Comment (Adityapnn811) [reply svelte/src/routes/test/scripts/converter/index.ts:151]: thanks

**Review by Adityapnn811 — APPROVED** [review]

**Pull request merged**